src/graph/test/SessionTest.cpp

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/test/TestEnv.h"
+#include "graph/test/TestBase.h"
+#include "graph/test/TraverseTestBase.h"
+#include "meta/test/TestUtils.h"
+
+namespace nebula {
+namespace graph {
+
+class SessionTest : public TestBase {
+protected:
+    void SetUp() override {
+        TestBase::SetUp();
+        // ...
+    }
+
+    void TearDown() override {
+        // ...
+        TestBase::TearDown();
+    }
+};
+
+TEST_F(SessionTest, ShowSessions) {
+    auto client = gEnv->getClient();
+    ASSERT_NE(nullptr, client);
+    {
+        cpp2::ExecutionResponse resp;
+        auto *fmt = "SHOW SESSIONS";
+        auto query = fmt;
+        auto code = client->execute(query, resp);
```

Comment:
checkout the result, such as the number of the clients and sessionid

Reply:
Good.

Reply:
session id is a changing value and should not be displayed